src/aap_eda/settings/default.py

```diff
@@ -139,7 +139,7 @@ def _get_secret_key() -> str:
 def _get_boolean(name: str, default=False) -> bool:
     value = settings.get(name, default)
     if isinstance(value, str):
-        value = str_to_bool(value)
+        value = utils.str_to_bool(value)
```

Comment:
@Alex-Izquierdo Is this related to this PR?

Reply:
I just import utils package for readability and usability, instead of importing functions one by one.